packages/fxa-auth-server/scripts/write-emails-to-disk.js

```diff
@@ -123,6 +123,7 @@ function sendMail(mailer, messageToSend) {
       country: 'Spain',
     },
     locations: [],
+    numberRemaining: 2,
```

Comment:
❤️ - thank you, I used the script to check the titles!

Reply:
Ha, I leaned on it a lot recently for #1791 too, so it was at the front of mind.